index.html

```diff
@@ -413,55 +413,77 @@ <h3>Proofs</h3>
+specified as a string that maps to a URL [[URL]]. Examples of proof types
+include <code>DataIntegrityProof</code> and <code>Ed25519Signature2020</code>.
+Proof types determine what other mandatory fields are required to secure and
+verify the proof.
           </dd>
 
-          <dt><dfn class="lint-ignore">proofPurpose</dfn><dt>
+          <dt><dfn class="lint-ignore">proofPurpose</dfn></dt>
           <dd>
-<em>Required</em>. The specific intent for the proof, the reason why an entity
-created it. Acts as a safeguard to prevent the proof from being misused for a
-purpose other than the one it was intended for. For example, a proof can be used
-for purposes of <code>authentication</code>, for asserting control of a
-Verifiable Credential (<code>assertionMethod</code>), and several others.
+The reason the proof was created MUST be specified as a string that maps to a
+URL [[URL]]. The proof purpose acts as a safeguard to prevent the proof from
+being misused for a purpose other than the one that was intended. For example, a
+proof can be created for the purpose of <code>authentication</code> or for
+asserting a set of statements such as those in a Verifiable Credential
+(<code>assertionMethod</code>).
           </dd>
 
-          <dt><dfn>verificationMethod</dfn><dt>
+          <dt><dfn>verificationMethod</dfn></dt>
           <dd>
-<em>Required</em>. A set of parameters required to independently verify the
-proof, such as an identifier for cryptographic material that would be used in
-the proof.
+The means and information needed to verify the proof MUST be specified as a
+string that maps to a [[URL]]. An example of a verification method is a
+link to a public key which includes crypographic material that is used by a
+verifier during the verification process.
           </dd>
 
-          <dt><dfn>created</dfn><dt>
+          <dt><dfn>created</dfn></dt>
           <dd>
-<em>Required</em>. The string value of an [[ISO8601]] combined date and time
-string generated by the <a href="#create-proof-algorithm">Proof Algorithm</a>.
+The date and time the proof was created MUST be specified as an
+[[XML-SCHEMA11]] combined date and time string.
           </dd>
 
-          <dt>domain<dt>
+          <dt>domain</dt>
           <dd>
-Optional. A string value specifying the restricted <a>domain</a> of the proof.
+A string value that SHOULD be included by the creator of a proof and used by a
+verifier to ensure the proof was intended to by used by the verifier. The
+specification of the `domain` parameter is useful in challenge-response
+protocols where the verifier is operating from within a security domain known to
+the creator of the proof. Examples of a domain parameter include:
+`domain.example` (DNS domain), `mycorp-intranet` (well-known text string), and
+`b31d37d4-dd59-47d3-9dd8-c973da43b63a` (UUID).
           </dd>
 
-          <dt><dfn>proofValue</dfn><dt>
+          <dt><dfn class="lint-ignore">nonce</dfn></dt>
```

Comment:
This definition is for a challenge. The `nonce` term is also defined, but is used to improve unlinkability when generating signal values (by randomizing them). It may be better to include the `nonce` within the `proofValue`, however, using whatever format best suites the cryptosuite -- so maybe we want a separate section for `nonce` that says that.

```suggestion
          <dt><dfn class="lint-ignore">challenge</dfn></dt>
```